testing/enchant2/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=enchant2
+pkgver=2.2.3
+pkgrel=0
+pkgdesc="A wrapper library for generic spell checking (v2)"
+url="https://abiword.github.io/enchant/"
+arch="all"
+license="LGPL-2.1-or-later"
+subpackages="$pkgname-dev $pkgname-doc"
+makedepends="aspell-dev glib-dev dbus-glib-dev hunspell-dev gettext-dev file
+	bash aspell-dev"
+options="!check" # needs unpackaged unittest-cpp
+source="https://github.com/AbiWord/enchant/releases/download/v${pkgver}/enchant-${pkgver}.tar.gz"
+builddir="$srcdir/enchant-$pkgver"
+
+build() {
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--disable-static
+	make
+}
+
+check() {
+	make check
+}
+
+package() {
+	make DESTDIR="$pkgdir" install
+	rm "$pkgdir"/usr/share/enchant/enchant.ordering
```

Comment:
enchant and enchant2 both provide this file, will have to look into how I can share this between them.

Reply:
Fixed.